scripts/package_release.sh

```diff
@@ -12,15 +12,15 @@ version=$1
 chartdir=$2
 values=$3
 
-charttmpdir=`mktemp -d 2>/dev/null || mktemp -d -t 'charttmpdir'`
+charttmpdir=$(mktemp -d 2>/dev/null || mktemp -d -t charttmpdir)
 
 charttmpdir=${charttmpdir}/chart
 
-cp -R deploy/chart/ ${charttmpdir}
-echo "Version: $1" >> ${charttmpdir}/Chart.yaml
+cp -R deploy/chart/ "${charttmpdir}"
+sed -i "s/^[Vv]ersion:.*\$/version: ${version}/" "${charttmpdir}/Chart.yaml"
```

Comment:
We may have compatibility issues for macos users that don't have gsed. Historically, this has plagued us as a team.

Reply:
Yes, I know I always ended up putting GNU utils at the front of my PATH when using a Mac in the past. In this case, I hope that we can sidestep the problem by automating release generation to run in a container/VM.